alot/commands/utils.py

```diff
@@ -90,15 +90,16 @@ def _get_keys(ui, encrypt_keyids, block_error=False, signed_only=False):
                                  signed_only=signed_only)
         except GPGProblem as e:
             if e.code == GPGCode.AMBIGUOUS_NAME:
-                tmp_choices = [k.uids[0].uid for k in
+                tmp_choices = [k.uids[0].uid + " (" + k.fpr + ")" for k in
```

Comment:
I really don't like string concatenation (it's slow on some python implementations, and it's hard to read), I think that `'{}({})'.format(k.uids[0].uid, k.fpr) would be more readable here.